src/coreclr/jit/emitarm64.cpp

```diff
@@ -6493,6 +6496,123 @@ void emitter::emitIns_R_R_R(
     appendToCurIG(id);
 }
 
+/*****************************************************************************
+ *
+ *  Add an instruction storing 2 registers into a memory (pointed by reg3) and the offset
+    (immediate).
+ */
+
+void emitter::emitIns_SS_R_R_R_I(instruction ins,
+                                 emitAttr    attr,
+                                 emitAttr    attr2,
+                                 regNumber   reg1,
+                                 regNumber   reg2,
+                                 regNumber   reg3,
+                                 ssize_t     imm,
```

Comment:
Here, `imm` represents the offset from `reg3` of the addressing mode. Normally, when load/store of a local, we don't need `imm` for this purpose (or `reg3`) because those are derived from the lclvar itself. Here we have a case where either the first or second slot stored, or both slots, are a lclvar. So we don't know the immediate to use for the base register offset.

However, we should at least be able to assert that `reg3` is the same register that varx1 and varx2 would choose (using FPbased return from `lvaFrameAddress`)

Reply:
done.